cmd/bosun/web/static/partials/rule.html

```diff
@@ -95,7 +95,7 @@
 </div>
 <div class="row" ng-show="error">
 	<div class="col-lg-12">
-		<div class="alert alert-danger" ng-bind="error"></div>
+		<pre class="alert alert-danger" ng-bind="error"></pre>
```

Comment:
This would stop line breaks for long lines. May need some CSS if we decide to stick with the pre.